libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -20,8 +20,34 @@
 #define AP_MOTORS_MOT_10 9U
 #define AP_MOTORS_MOT_11 10U
 #define AP_MOTORS_MOT_12 11U
-
+#define AP_MOTORS_MOT_13 12U
+#define AP_MOTORS_MOT_14 13U
+#define AP_MOTORS_MOT_15 14U
+#define AP_MOTORS_MOT_16 15U
+#define AP_MOTORS_MOT_17 16U
+#define AP_MOTORS_MOT_18 17U
+#define AP_MOTORS_MOT_19 18U
+#define AP_MOTORS_MOT_20 19U
+#define AP_MOTORS_MOT_21 20U
+#define AP_MOTORS_MOT_22 21U
+#define AP_MOTORS_MOT_23 22U
+#define AP_MOTORS_MOT_24 23U
+#define AP_MOTORS_MOT_25 24U
+#define AP_MOTORS_MOT_26 25U
+#define AP_MOTORS_MOT_27 26U
+#define AP_MOTORS_MOT_28 27U
+#define AP_MOTORS_MOT_29 28U
+#define AP_MOTORS_MOT_30 29U
+#define AP_MOTORS_MOT_31 30U
+#define AP_MOTORS_MOT_32 31U
+
+#ifndef AP_MOTORS_MAX_NUM_MOTORS
+#if AP_SCRIPTING_ENABLED
+#define AP_MOTORS_MAX_NUM_MOTORS 32
```

Comment:
I think this needs to be constrained to NUM_SERVO_CHANNELS. We lower this on certain boards to save space and it constrains the size of arrays in certain circumstances

Reply:
I've added another patch to constrain it to NUM_SERVO_CHANNELS - is this what you had in mind?

Reply:
LGTM